packages/evm/contracts/Hashi.sol

```diff
@@ -34,61 +34,89 @@
 pragma solidity ^0.8.17;
```

Comment:
I would recommend to require at least 0.8.20

Reply:
yes! i'll upgrade it with another pr!